zone/attack.cpp

```diff
@@ -2571,6 +2572,10 @@ bool NPC::Death(Mob* killer_mob, int32 damage, uint16 spell, EQ::skills::SkillTy
 		}
 
 		entity_list.RemoveFromAutoXTargets(this);
+
+		if (killer->GetUltimateOwner()->IsClient()) {
```

Comment:
We should check for valid pointer before accessing it

eg

`killer->GetUltimateOwner()`